docs/event-bus/08-01-configure-default-channelprovisioner.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Configure default ClusterChannelProvisioner
+type: Tutorials
+---
+
+Kyma comes with NATSS as its default ClusterChannelProvisioner.  You can see the configuration details in the [default-channel-webhook](../../resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml) ConfigMap . You can, however, use other default channel provisioner of your choice. An example can be the in-memory-channel provisioner.
```

Comment:
Remove the space between ConfigMap and the dot.

Reply:
done